src/components/core/wizard/mobile-wizard/step-navigation.tsx

```diff
@@ -75,14 +78,14 @@ const StepNavigation = React.memo(({ steps, current_step_index, dark, next_step_
     return (
         <StepContainer data-testid="step-navigation">
             <ProcessCircle style={{ background: `${progress_angel}` }}>
-                <Inner dark={dark}>
-                    <Text as="label" type="paragraph-1" bold style={{ color: dark ? '#FFFFFF' : '$greyLight700' }}>
+                <Inner dark={isDark}>
+                    <Text as="label" type="paragraph-1" bold style={{ color: isDark ? '#FFFFFF' : '$greyLight700' }}>
                         {current_step_number}/{total_steps_count}
                     </Text>
                 </Inner>
             </ProcessCircle>
             <div style={{ display: 'flex', flexDirection: 'column' }}>
-                <Text as="label" type="paragraph-1" bold style={{ color: dark ? '#FFFFFF' : '$greyLight700' }}>
+                <Text as="label" type="paragraph-1" bold style={{ color: isDark ? '#FFFFFF' : '$greyLight700' }}>
```

Comment:
Same

Reply:
I'm not very familiar with the component but I changed the implementation as you mentioned. thanks :pray: